src/Forms/FormScaffolder.php

```diff
@@ -110,7 +110,9 @@ public function getFieldList()
         }
 
         // add has_one relation fields
-        if ($this->obj->hasOne()) {
+        if ($this->obj->hasOne()
```

Comment:
There's 4 variations of this condition check. I think it would probably be worthwhile moving it to a small private method. Something like :
```php
private function includeRelation($type) {
    return 
        $this->obj->$type() && 
        ($this->includeRelations === true || !empty($this->includeRelations[$type]);
}
```